wildfly-swarm/community/health-check/booster.yaml

```diff
@@ -6,7 +6,7 @@ source:
     ref: master
 metadata:
   runsOn:
-    - '!osio'
+    - 'osio'
```

Comment:
I wonder if it will be _only_ OSIO with this change. I.e., no Starter, no Pro, no nothing. @gastaldi?

Reply:
Good point, we don't want this change to make it not work on OSO Starter and Pro, or even local.

How does the filtering work?

Reply:
This tells the booster to run only in OSIO, the solution is to remove the metadata tag. The PR is wrong, I'm reverting this commit